osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -276,13 +275,13 @@ protected virtual void BindDrawableRuleset(DrawableRuleset drawableRuleset)
             Progress.BindDrawableRuleset(drawableRuleset);
         }
 
-        protected virtual SkinnableAccuracyCounter CreateAccuracyCounter() => new SkinnableAccuracyCounter();
+        protected SkinnableAccuracyCounter CreateAccuracyCounter() => new SkinnableAccuracyCounter();
 
-        protected virtual SkinnableScoreCounter CreateScoreCounter() => new SkinnableScoreCounter();
+        protected SkinnableScoreCounter CreateScoreCounter() => new SkinnableScoreCounter();
 
-        protected virtual SkinnableComboCounter CreateComboCounter() => new SkinnableComboCounter();
+        protected SkinnableComboCounter CreateComboCounter() => new SkinnableComboCounter();
 
-        protected virtual SkinnableHealthDisplay CreateHealthDisplay() => new SkinnableHealthDisplay();
+        protected SkinnableHealthDisplay CreateHealthDisplay() => new SkinnableHealthDisplay();
```

Comment:
These will eventually be dismantled; just leaving in place for the time being (need to plan where these "defaults" will sit going forward).